src/network/networkd-network.c

```diff
@@ -405,6 +405,7 @@ int network_load_one(Manager *manager, OrderedHashmap **networks, const char *fi
                 .dhcp_broadcast = -1,
                 .dhcp_ipv6_only_mode = -1,
                 .dhcp_6rd_prefix_route_type = RTN_UNREACHABLE,
+                .dhcp_send_bootp = false,
```

Comment:
no need to initialize this, unspecified fields are zero-initialized anyway. please drop